File: Bars.Gkh/Entities/EntityHistory/EntityHistoryField.cs
namespace Bars.Gkh.Entities
{
    using Bars.B4.DataAccess;

    /// <summary>
    /// Логируемое поле сущности
    /// </summary>
    public class EntityHistoryField : PersistentObject
    {
        /// <summary>
        /// Старое значение
        /// </summary>
        public virtual string OldValue { get; set; }

        /// <summary>
        /// Новое значение
        /// </summary>
        public virtual string NewValue { get; set; }

        /// <summary>
        /// Наименование поля
        /// </summary>
        public virtual string FieldName { get; set; }

        /// <summary>
        /// Информация об изменении
        /// </summary>
        public virtual EntityHistoryInfo EntityHistoryInfo { get; set; }
    }
}

File: Bars.Gkh/libs/B4/store/EntityHistoryField.js
Ext.define('B4.store.EntityHistoryField', {
    extend: 'B4.base.Store',
    autoLoad: false,
    sortOnLoad: true,
    proxy: {
        type: 'b4proxy',
        controllerName: 'EntityHistoryField',
        listAction: 'list',
        createAction: '',
        getAction: '',
        updateAction: '',
        deleteAction: ''
    },
    fields: [
        { name: 'Id' },
        { name: 'EntityHistoryInfo' },
        { name: 'FieldName' },
        { name: 'OldValue' },
        { name: 'NewValue' }
    ],
    sorters: [
        {
            property: 'Id',
            direction: 'DESC'
        }
    ]
});